_docs/resources/index.md

```diff
@@ -1,7 +1,120 @@
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/palette-contrast.svg" alt="color palette icon"/>
+    <h4>Palette contrast checker</h4>
+  </div>
+    <p >When finalizing color schemes for a component or layout, accessibility is always a concern. This lightweight browser app lets you select a color from the Helix palette and check readability against different backgrounds.</p>
+    <a id="link" target="_blank" href="http://citguy.com/PaletteContrast">View the tool</a>
+</div>
+
+{% endcolumn %}
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/codepen.svg" alt="codepen logo icon"/>
+    <h4>Codepen Template</h4>
+  </div>
+    <p >If you are looking to test out HelixUI web components, look no further than this pen. The Helix team has created a codepen template that requires all the necesary polyfills and HelixUI code for quick and easy testing.</p>
+    <a id="link" target="_blank" href="https://codepen.io/pen?template=YaRqQg">Create a pen</a>
+</div>
+
+{% endcolumn %}
+
+</div>
+</section>
+
+<section class="static-section" markdown="1">
+
+## GitHub Repos
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/mark-github.svg" alt="GitHub logo icon"/>
+    <h4>Design system repo</h4>
+  </div>
+    <p>The repository is used to generate the Helix documentation site (this site). Feel free to star, submit issues, or help. Note: You will need access to the rackerlabs organization on public GitHub to view.</p>
+    <a id="link" target="_blank" href="https://github.com/rackerlabs/design-system">View the repo</a>
+</div>
+
+{% endcolumn %}
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/mark-github.svg" alt="GitHub logo icon"/>
+    <h4>HelixUI repo</h4>
+  </div>
+    <p>HelixUI is the public open-source implementation of the Helix language built using web components. </p>
+    <a id="link" target="_blank" href="https://github.com/rackerlabs/helix-ui">View the repo</a>
+</div>
+
+{% endcolumn %}
+
+{% column left:"hxCol-6 hxCol-xs-12 hxCol-sm-12 hxCol-md-6 hxCol-lg-6" %}
+
+<div class="resource-card">
+  <div class="icon"><img src="{{site.baseurl}}/assets/images/mark-github.svg" alt="GitHub logo icon"/>
+    <h4>hxSketchLibrary repo</h4>
+  </div>
+    <p>The hxSketchLibrary repo is used to generate and maintain the Sketch Library. It deconstructs the library into nested json structures with a program called kactus.io. If you notice something amiss in the library, feel free to submit issues or pull requests here.</p>
```

Comment:
recommend removing the first sentence, and mentioning some specifics:

To manage Sketch assets in GitHub we use [Kactus](https://kactus.io/), which converts Sketch files to JSON for easier storage.  If you want to contribute to the Helix library, you must use Kactus to work with the repo.  If you just the latest library file, we have included the most recent, generated Sketch files in the repo too.